noxfile.py

```diff
@@ -38,7 +38,7 @@
     "unit",
     "unit_grpc_gcp",
     "unit_wo_grpc",
-    "unit_with_auth_aio",
+    "unit_with_async_rest_extra",
```

Comment:
`"unit_w_async_rest_extra"`

Reply:
Fixed in https://github.com/googleapis/python-api-core/pull/701/commits/9f7d36c90ed2f4f287a9305e6b18431f04d616c6